<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessListClass</class>
 <widget class="QWidget" name="ProcessListClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>处理列表</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCheckBox" name="cB_addFog">
     <property name="text">
      <string>加雾</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gB_addFog">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>加雾</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>仿真雾浓度系数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="fog_para"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cB_addRain">
     <property name="text">
      <string>加雨</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gB_addRain">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>加雨</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>雨滴文件路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rain_path"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>选择文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>相对雨量系数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="rain_para"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cB_addSnow">
     <property name="text">
      <string>加雪</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gB_addSnow">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>加雪</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>相对雪量系数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="snow_para"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pB_do">
       <property name="text">
        <string>确认修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_reset">
       <property name="text">
        <string>还原图像</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cB_addFog</sender>
   <signal>toggled(bool)</signal>
   <receiver>gB_addFog</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cB_addRain</sender>
   <signal>toggled(bool)</signal>
   <receiver>gB_addRain</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cB_addSnow</sender>
   <signal>toggled(bool)</signal>
   <receiver>gB_addSnow</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
